src/main/java/io/kestra/plugin/notifications/mail/MailSend.java

```diff
@@ -167,14 +175,15 @@ public VoidOutput run(RunContext runContext) throws Exception {
         logger.debug("Sending an email to {}", to);
 
         final String htmlContent = runContext.render(this.htmlTextContent);
+        final String textContent = runContext.render(this.plainTextContent);
 
         // Building email to send
         EmailPopulatingBuilder builder = EmailBuilder.startingBlank()
             .to(runContext.render(to))
             .from(runContext.render(from))
             .withSubject(runContext.render(subject))
             .withHTMLText(htmlContent)
-            .withPlainText("Please view this email in a modern email client")
+            .withPlainText(textContent)
```

Comment:
```suggestion
            .withPlainText(textContent != null ? textContent : "Please view this email in a modern email client")
```

As textContent is not mandatory, better to keep some default sentence in case it is not provided instead of having an empty content.

Reply:
@loicmathieu correct me if I am wrong:
Seems tjat `htmlContent` also not mandatory based on documentation, but expected that should be come from tempalte if I understood correct. And that was the reason why I also added templates with default texts `"Please view this email in a modern email client"`. 
if anyway here can be empty or null, the same can happen with `htmlContent`, isn't it ?

Reply:
Yes, but as we previously have this functionality, let's keep it.

Reply:
Handled https://github.com/kestra-io/plugin-notifications/pull/139/commits/d84fa92be782dbdafc72662731f539022174ba06